platform/ios/CHANGELOG.md

```diff
@@ -59,6 +59,8 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 * Added haptic feedback that occurs when the user rotates the map to due north, configurable via `MGLMapView.hapticFeedbackEnabled`. ([#10847](https://github.com/mapbox/mapbox-gl-native/pull/10847))
 * Added `MGLMapView.showsScale` as the recommended way to show the scale bar. This property can be set directly in Interface Builder. ([#11335](https://github.com/mapbox/mapbox-gl-native/pull/11335))
 * Fixed an issue where the scale bar would not appear until the map had moved. ([#11335](https://github.com/mapbox/mapbox-gl-native/pull/11335))
+* Fixed an issue where removing a `MGLOpenGLStyleLayer` from a map might result in a crash. ([#11553](https://github.com/mapbox/mapbox-gl-native/pull/11553))
+* Added `MGLMapPoint`, `MGLMapPointMake`, and `MGLStyleLayerDrawingContext.projectionMatrix` to more easily project `CLLocationCoordinate2D`s to screen points when rendering with `MGLOpenGLStyleLayer`. ([#10786](https://github.com/mapbox/mapbox-gl-native/pull/10786))
```

Comment:
In https://github.com/mapbox/mapbox-gl-native/pull/10786#pullrequestreview-109828412, I was suggesting that this function be added to the changelog, not the undocumented `MGLStyleLayerDrawingContext.projectionMatrix` field.